dgraph/cmd/bulk/run.go

```diff
@@ -70,7 +70,7 @@ func init() {
 	flag.Int("shufflers", 1,
 		"Number of shufflers to run concurrently. Increasing this can improve performance, and "+
 			"must be less than or equal to the number of reduce shards.")
-	flag.Bool("version", false, "Prints the version of dgraph-bulk-loader.")
+	flag.Bool("version", false, "Prints the version of Dgraph.")
```

Comment:
Dgraph Bulk Loader.

If we just say Dgraph, a user could make it mean Dgraph Alpha or Zero version, which might be different from the bulk loader.

Reply:
Isn't the version all the same across all the subcommands, though?